themes/bootstrap5/js/account_ajax.js

```diff
@@ -197,6 +238,9 @@ VuFind.register('account', function Account() {
     }
   };
 
+  /**
+   * Initialize elements which should clear the account cache
```

Comment:
Maybe this should be more general, like "Initialize the account AJAX system," since this is the general initialization hook and might have other functionality added someday (in theory).